containers/Investment/RevolvingPool/View/EpochOverview.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react'
+import { Box, Button, Heading, Table, TableBody, TableRow, TableCell } from 'grommet'
+import { ITinlake as ITinlakeV3 } from '@centrifuge/tinlake-js-v3'
+import { createTransaction, useTransactionState, TransactionProps } from '../../../../ducks/transactions'
+import { connect } from 'react-redux'
+
+interface Props extends TransactionProps {
+  tinlake: ITinlakeV3
+}
+
+const EpochOverview: React.FC<Props> = (props: Props) => {
+  const [epochId, setEpochId] = React.useState(0)
+  // const [epochMinimumTimeEnd, setEpochMinimumTimeEnd] = React.useState(0)
+  const [epochState, setEpochState] = React.useState('')
+
+  // const endHoursFromNow = Math.floor((epochMinimumTimeEnd - new Date().getTime()) / 1000 / 60 / 60)
+
+  React.useEffect(() => {
+    async function getState() {
+      setEpochId(await props.tinlake.getCurrentEpochId())
+      // setEpochMinimumTimeEnd(await props.tinlake.getCurrentEpochMinimumTimeEnd())
+      setEpochState(await props.tinlake.getCurrentEpochState())
+    }
+
+    getState()
+  }, [])
```

Comment:
Should this be run every second/in another interval?

Reply:
Done!